README.md

```diff
@@ -298,7 +309,65 @@ OPTIONS
   --help                 show CLI help
 ```
 
-_See code: [src/commands/instance/spawn.ts](https://github.com/kuzzleio/kourou/blob/v0.5.0/src/commands/instance/spawn.ts)_
+_See code: [src/commands/instance/spawn.ts](https://github.com/kuzzleio/kourou/blob/v0.6.0/src/commands/instance/spawn.ts)_
+
+## `kourou vault:add SECRETS-FILE KEY VALUE`
+
+Add an encrypted key to a secrets file
```

Comment:
Little typo here at "secrets" ?

Reply:
Hmm I'm not sure it's a typo since it's not a "secret file" (AKA a file that people don't know about) but a "secrets file" (AKA a file containing many secrets)